Formula/dps8m.rb

```diff
@@ -1,14 +1,16 @@
 class Dps8m < Formula
-  desc "Simulator for the Multics dps-8/m mainframe"
-  homepage "https://ringzero.wikidot.com/"
-  url "https://gitlab.com/dps8m/dps8m/-/archive/R3.0.0/dps8m-R3.0.0.tar.gz"
-  sha256 "cee3cc0942ceb929d267c4989551526ff50e777706f62bfc449e16d1428a1d2e"
+  desc "Simulator of the 36‑bit GE/Honeywell/Bull 600/6000‑series mainframe computers"
+  homepage "https://dps8m.gitlab.io/"
+  url "https://dps8m.gitlab.io/dps8m-archive/R3.0.0/dps8m-r3.0.0-src.tar.gz"
+  sha256 "e3eac9e4f8b6c7fad498ff1848ba722e1a2e220b793ce02e2ea6a7a585e0c91f"
   license "ICU"
+  revision 1
   head "https://gitlab.com/dps8m/dps8m.git", branch: "master"
 
   livecheck do
-    url :stable
-    regex(/^R?(\d+(?:\.\d+)+)$/i)
+    url "https://dps8m.gitlab.io/dps8m/"
+    regex(/DPS8M R?(\d+(?:\.\d+)+)</i)
+    strategy :page_match
```

Comment:
```suggestion
    url "https://dps8m.gitlab.io/dps8m/Releases/"
    regex(/href=.*?dps8m[._-]r?(\d+(?:\.\d+)+)[._-]src\.t/i)
```

The releases page links to the `stable` tarball we're using in the formula, so it's preferable to match the version from that.

For what it's worth, we only use `#strategy` when we need to override the strategy that livecheck selects for a given URL (if any) or if we need to use a `strategy` block. In this case, livecheck is already using `PageMatch` for this URL, so the `#strategy` call isn't necessary.

Reply:
Great, thanks!